backend/app/utils/database.py

```
@@ -0,0 +1,78 @@
+import os
+import asyncio
+import traceback
+from typing import List, Dict, Any, Optional
+from ..config import Config
+from .logger import get_logger
+
+logger = get_logger('mirofish.database')
+
+def _get_env():
```

Ruff (0.15.11)

[warning] Missing return type annotation for private function `_get_env`

(ANN202)